docs/topic/billing/chargeable-resources.md

```diff
@@ -0,0 +1,621 @@
+   the cheapest (hence slowest disk) possible.
+
+2. Prometheus disk
+
+   We use [prometheus](https://prometheus.io/) to store time series data,
+   both for operational (debugging) as well as reporting (usage metrics)
+   purposes. We retain 1 year of data, and that requires some disk space.
+   We try to not use the slowest disk type, but this doesn't require the
+   fastest one either.
+
+   Currently, most prometheus disks are bigger than they need to be. We
+   can slim them down with effort if needed.
+
+3. Grafana disk
+
+   Grafana also uses a sqlite database the same way JupyterHub does. While
+   most of the dashboards we use are [managed in an external repo](https://github.com/jupyterhub/grafana-dashboards),
+   the database still holds user information as well as manually set up
+   dashboards. This does not need to be fast, and we also try to have the
+   smallest possible disk here (1GiB)
+
+4. Optional per-user disk for dedicated databases
+
+   One of the features we offer is a database per user, set up as a sidecar.
+   Most databases do not want to use NFS as the backing disk for storing
+   their data, and need a dedicated block storage device. One is created
+   *for each user* (via [Kubernetes Dynamic Volume Provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/)),
+   and this can get quite expensive
+   (for reasons outlined in the ["Home Directory Storage" section](topic:billing:resources:home)).
+
+Pricing is primarily dependent on the expected disk performance (measured as throughput and IOPS) as well as the disk size. While there's a lot of
+options available, we'll list what we most commonly use in this table.
+
+| Cloud Provider | Cost | Pricing Page |
+| - | - | - |
+| GCP | Standard (Slow) - $0.048 / GB / month, Balanced (Most common) - $0.12 / GB / month  | [link](https://cloud.google.com/compute/disks-image-pricing#disk) |
+| AWS | gp2 (most common) - $0.1 / GB / month | [link](https://aws.amazon.com/ebs/pricing/) |
+| Azure | ??? | [link](https://azure.microsoft.com/en-us/pricing/details/managed-disks/) |
+
+
+## Kubernetes Cluster management
+
+Cloud providers charge for maintaining the Kubernetes cluster itself,
+separate from the nodes used to run various things in it. This is usually a *fixed* per hour cost per cluster.
+
+1. [AWS EKS](https://aws.amazon.com/eks/pricing/)
+
+   Costs $0.10 per hour flat.
+
+2. [GCP GKE](https://cloud.google.com/kubernetes-engine/pricing#standard_edition)
+
+   Is a bit messier - it costs $0.10 per hour, but is structured in such a way
+   that one single-zone (non highly available) cluster is free per GCP project.
+   While this is a cost saving of about 75$ per month, in most cases we
+   use *regional* (multi-zonal) clusters that are not free. This makes
+   master upgrades much more painfree, and reduces overall amount of downtime
+   the hub has. So our default here is to pay the 75$ a month for the
+   regional cluster.
```

Comment:
```suggestion
   Costs $0.10 per hour flat.
   
   There is a discount of about 75$ per month allowing one *zonal* (single zone, non highly available) cluster per GCP billing account continuously for free. This discount is of little value to us and communities since since we favor the *regional* (multi-zone, highly available) clusters for less disruptive kubernetes control plane upgrades, and since we don't manage that many separate billing accounts.
```